src/index.mjs

```diff
@@ -10,22 +11,10 @@ import stripIndent from 'strip-indent';
 
 const DEFAULT_CONFIG_FILE = '.stylelintrc.json';
 
-const STYLELINT_CONFIG_PATHS = new Set([
-	'.stylelintrc',
-	'.stylelintrc.cjs',
-	'.stylelintrc.json',
-	'.stylelintrc.yaml',
-	'.stylelintrc.yml',
-	'.stylelintrc.js',
-	'stylelint.config.js',
-	'stylelint.config.cjs',
-]);
+function getExistingConfigsInDirectory() {
```

Comment:
[suggest] How about renaming the function name because the function no longer returns an array?

```suggestion
function getExistingConfigInDirectory() {
```